src/pretix/settings.py

```diff
@@ -116,6 +116,28 @@
 
 db_options = {}
 
+USE_DATABASE_TLS = config.has_option('database', 'tls')
+USE_DATABASE_MTLS = config.has_option('database', 'mtls')
```

Comment:
I don't see either of these in the documentation. I also don't think they are necessary, couldn't we just derive them from sslmode?

Reply:
>I don't see either of these in the documentation. 

You're right is not documented, because it was my initial idea. I prefer in general the solution of using the ca cert for TLS and the corresponding client cert for mTLS as switching indicator.

Reply:
I'm not sure I agree (but I also don't have strong feelings about it).

If I configure

```
sslmode=disable
sslcert=/etc/cert.pem
```

I would expect TLS **not** to be used, wouldn't it?

Reply:
Yes, you are right. I didn't think that there might be a case where you don't even want to use verify-ca.